### PR TITLE
test  commit

### PR DESCRIPTION
**Pull request by robert1106**
hi Bro!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,5 +11,6 @@
 </head>
 <body>
   <app-root></app-root>
+  <p>Hi Maxim</p>
```
Comment (MaximRobota) [inline src/index.html:14]: It is not sum function!!! 
please add function sum(a, b) { ....

**Comment by robert1106** [comment]
new commit

File: src/app/blog/blog.component.ts [hunk 2]
```diff
@@ -156,7 +160,10 @@ export class BlogComponent implements OnInit {
 
 
   ngOnInit(): void {
-
```
Comment (MaximRobota) [inline src/app/blog/blog.component.ts:159]: Please add function in body component and call from ngOnInit

Comment (MaximRobota) [reply src/app/blog/blog.component.ts:159]: дщд

**Review by MaximRobota — APPROVED** [review]

**Comment by MaximRobota** [comment]
123

**Comment by MaximRobota** [comment]
cool

**Pull request merged**